src/libraries/Microsoft.PowerFx.Json/FormulaValueJSON.cs

```diff
@@ -107,7 +107,11 @@ public static FormulaValue FromJson(JsonElement element, FormulaValueJsonSeriali
                     {
                         DateTime dt3 = element.GetDateTime();
                         return DateTimeValue.New(TimeZoneInfo.ConvertTimeToUtc(dt3));
-                    }                              
+                    }
+                    else if (formulaType is BlobType)
+                    {
+                        return FormulaValue.NewBlob(element.GetBytesFromBase64());
+                    }

```

Comment:
Since this is shared code, can we ensure this is not breaking JSON() function w.r..t Power Apps?